drizzlepac/hlautils/align_utils.py

```diff
@@ -195,8 +196,11 @@ def get_fit_methods(self):
         """
         return list(self.fit_methods.keys())
 
-    def perform_fit(self, method_name, catalog_name, reference_catalog):
+    def perform_fit(self, method_name, catalog_name, reference_catalog, 
+                    fitgeom='general'):
         """Perform fit using specified method, then determine fit quality"""
+        # Updated fits_pars with value for fitgeom
+        self.fit_pars[method_name]['fitgeom'] = fitgeom
```

Comment:
There should be an informational message here that a value in a parameter file driving functionality has been overridden by a new value.

Reply:
Good idea.   Implemented in e78adcc.